app/views/rooms/_form.html.erb

```diff
@@ -156,6 +168,19 @@
 			<%= t('default.room.usesharedcode') %>&nbsp;
 			<br>
 		</div>
+    
+    <div>
+      <p class="font-sans inline-block font-light"> <%= "#{t('default.room.sharedCode.numberOfRooms')} #{@room.count_by_shared_code}" %> </p>
+      <%# only show revoke button if there are other rooms using the code. %>
+      <% if @room.count_by_shared_code > 0 %>
+        <%= button_tag t('default.room.sharedCode.revokeCode'),
+            type: 'button',
+            id: 'revoke-code-btn', 
+            data: { url: revoke_shared_code_path(@room, :launch_nonce => @launch_nonce), confirm: t('default.room.sharedCode.revokeCodeAlert') }, 
+            class: 'font-sans text-center text-red-600 border border-red-600 mb-0 bg-white px-4 py-2 rounded hover:text-white hover:bg-red-600' %>
```

Comment:
If you used the `button_to` here, you can link it directly to the action on the rails side, without having to add the Javascript